src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefault.java

```diff
@@ -19,4 +19,27 @@ public void foo() {
         case 2: i--; break;
         }
     }
+
+    public void nestedSwitch1() {
```

Comment:
please add `// violation` comments on lines with violation

Reply:
Done, I have added comments only in unitTest inputs.

Reply:
Updated the Diff report.